schneiderWallbox/schneiderwallbox-registers.json

```diff
@@ -0,0 +1,498 @@
+            "address": 9,
+            "size": 1,
+            "type": "uint16",
+            "enum": "LastChargeStatus",
+            "registerType": "holdingRegister",
+            "readSchedule": "update",
+            "description": "lastChargeStatus",
+            "defaultValue": "LastChargeStatusOk",
+            "access": "RO"
+        },
+        {
+            "id": "remoteCommandStatus",
+            "address": 20,
+            "size": 1,
+            "type": "uint16",
+            "registerType": "holdingRegister",
+            "readSchedule": "update",
+            "description": "remoteCommandStatus",
+            "defaultValue": "0",
+            "access": "RO"
+        },
+        {
+            "id": "errorStatus",
+            "address": 23,
+            "size": 2,
+            "type": "uint32",
+            "registerType": "holdingRegister",
+            "readSchedule": "update",
+            "description": "errorStatus",
+            "defaultValue": "0",
+            "access": "RO"
+        },
+        {
+            "id": "chargeTime",
+            "address": 30,
+            "size": 2,
+            "type": "uint32",
+            "registerType": "holdingRegister",
+            "readSchedule": "update",
+            "description": "chargeTime",
+            "defaultValue": "0",
+            "access": "RO"
+        },
+        {
+            "id": "remoteCommand",
+            "address": 150,
+            "size": 1,
+            "type": "uint16",
+            "enum": "RemoteCommand",
+            "registerType": "holdingRegister",
+            "readSchedule": "update",
+            "description": "remoteCommand",
+            "defaultValue": "RemoteCommandAcknowledgeCommand",
+            "access": "RW"
+        },
+        {
+            "id": "maxIntensitySocket",
+            "address": 301,
+            "size": 1,
+            "type": "uint16",
+            "registerType": "holdingRegister",
+            "readSchedule": "update",
+            "description": "maxIntensitySocket",
+            "defaultValue": "0",
+            "unit": "Ampere",
+            "access": "RW"
+        },
+        {
+            "id": "maxIntensityCluster",
+            "address": 310,
+            "size": 1,
+            "type": "uint16",
+            "registerType": "holdingRegister",
+            "readSchedule": "update",
+            "description": "maxIntensityCluster",
+            "defaultValue": "0",
+            "unit": "Ampere",
+            "access": "RW"
+        },
+        {
+            "id": "remoteControllerLifeBit",
+            "address": 932,
+            "size": 1,
+            "type": "uint16",
+            "registerType": "holdingRegister",
+            "readSchedule": "update",
+            "description": "remoteControllerLifeBit",
+            "defaultValue": "0",
+            "access": "RW"
+        },
+        {
+            "id": "degradedMode",
+            "address": 933,
+            "size": 1,
```

Comment:
this and the previous one could be put into a block for micro-performance-optimization (reading a block of 2 is faster then reading 2 registers individually). That said, it depends mostly on the device how fast/slow it is to read registers... If the schneider one is very fast, the optimization would be negligible, but we've seen devices where a single read call takes some 300ms, in which case this would a reasonable big optimization.

Again, just a hint, not forcing a change here.